### PR TITLE
[ZH] Add `Mascots`

### PR DESCRIPTION
**Pull request by YumeMuzi**

### CODE REVIEW

File: wiki/Mascots/zh.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Yuzu 是 osu!catch 游戏模式的吉祥物，于 2014 年 6 月 22 日加入。他出生于 2000 年 4 月 10 日，身高 172 厘米，体重 65 千克。他现在的形象由 [Thievley](https://osu.ppy.sh/users/4717672) 设计，而他最初的艺术形象和接水果小人是由 [ztrot](https://osu.ppy.sh/users/6347) 设计。而 Daru 设计了 comboburst 时出现在屏幕上的形象。
+
+### ![osu!mania icon](/wiki/shared/mode/mania.png) Mani & Mari
+
+*获取更多信息，参见：[Introducing Mani and Mari, the New osu!mania Mascots.](https://osu.ppy.sh/home/news/2020-09-17-introducing-mani-mari-osumania)*
+
+![Mani & Mari](https://assets.ppy.sh/media/mari-mani/wiki-key-condensed.png "Mani & Mari")
+
+Mani 和 Mari 是由 [xiemon](https://osu.ppy.sh/users/5203667) 设计，在 [Most Manic Art Contest](https://osu.ppy.sh/community/contests/80) 中脱颖而出，并于 2020 年 9 月 18 日成为 osu!mania 模式的官方吉祥物。Mani 和 Mari 是互相看不惯对方，但又不可否认地很相似的一对双胞胎。
```
Comment (kj415j45) [inline wiki/Mascots/zh.md:29]:
```suggestion
Mani 和 Mari 由 [xiemon](https://osu.ppy.sh/users/5203667) 设计，在 [Most Manic Art Contest](https://osu.ppy.sh/community/contests/80) 中脱颖而出，并于 2020 年 9 月 18 日成为 osu!mania 模式的官方吉祥物。Mani 和 Mari 是互相看不惯对方，但又不可否认地很相似的一对双胞胎。
```
第一句的层次感觉还是有些怪，可能需要调整语序？

File: wiki/Mascots/zh.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Mani 是个特立独行的人，他总是渴望探索新花样和新事物。同时，他的妹妹 Mari（以前又被叫做 Maria）是一个传统、死板的理想主义者，同时也很喜欢成为聚光灯下的焦点。这对双胞胎的性格截然不同，很难相处。
```
Comment (kj415j45) [inline wiki/Mascots/zh.md:31]:
```suggestion
Mani 是个特立独行的人，他总是渴望探索新花样和新事物。同时，他的妹妹 Mari（以前又被叫做 Maria）是一个传统、保守的理想主义者，也希望成为聚光灯下的焦点。因性格迥异，他们之间水火不相容。
```
“死板”感觉偏贬义了一些，“保守”较为中性。

File: wiki/Mascots/zh.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+## 客串
+
+### 一本木龙太
+
+![Ryuuta](img/Ryuuta.png "Ryuuta")
+
+> 他是应援团中的热血头儿，有一颗善良的心，总是为周围陷入困难与窘境的人们加油打劲！
+
+一本木龍太（Ryūta Ippongi）曾是 osu!catch 模式中的接水果小人，于 2008 年加入 osu!，但于 2014 年被 [Yuzu](#-yuzu) 替代。他的艺术形象由 [iNiS 公司](https://en.wikipedia.org/wiki/INiS) 设计，并曾作为老版官方网页的一部分。同时，他也曾在 [病娇模拟器](https://yanderesimulator.com) 中作为 NPC 出现。
```
Comment (kj415j45) [inline wiki/Mascots/zh.md:49]:
```suggestion
一本木龍太（Ryūta Ippongi）曾是 osu!catch 模式中的接水果小人，于 2008 年加入 osu!，但于 2014 年被 [Yuzu](#-yuzu) 替代。他的艺术形象由 [iNiS 公司](https://en.wikipedia.org/wiki/INiS) 设计，并曾作为老版官方网页的一部分。他也曾在 [病娇模拟器](https://yanderesimulator.com) 中作为 NPC 出现。
```

File: wiki/Mascots/zh.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+特工 J，又称为 BA-2 (节拍特工-2) 或 J，曾于 2008 年作为 osu! 吉祥物的一员，但随后于 2014 年被解雇。他的艺术形象也由 iNiS 公司设计，并也曾作为老版官方网页的一部分。
+
+特工 J 也曾出现在 [LuigiHann](https://osu.ppy.sh/users/1079) 设计的皮肤 [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357) 中。
+
+### Don
+
+![Don](img/Don.png "Don")
+
+> Don 是游戏 [太鼓达人](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) 系列的主角。他是一只有白色鼓边、红橙色鼓面、淡蓝色身体、四只脚的太鼓。Don 的梦想是把太鼓的美妙带给全世界。在他搬到和田屋三年之后，城镇中无人不晓他的名号。他食量惊人，经常花重金买许多好吃的，以至于和田屋有时候都负担不起他的消费。他的口癖是经常在句尾带上一句 "Ta-Don"，意为敲击鼓面的声音。
```
Comment (kj415j45) [inline wiki/Mascots/zh.md:67]:
```suggestion
> Don 是游戏 [太鼓达人](https://zh.wikipedia.org/wiki/%E5%A4%AA%E9%BC%93%E4%B9%8B%E9%81%94%E4%BA%BA%E7%B3%BB%E5%88%97) 系列的主角。他是一只有白色鼓边、红橙色鼓面、淡蓝色身体、四只脚的太鼓。Don 的梦想是把太鼓的美妙带给全世界。在他搬到和田屋三年后，城镇中无人不晓他的名号。他食量惊人，经常花重金买许多好吃的，以至于和田屋有时候都负担不起他的消费。他的口癖是经常在句尾带上一句 "Ta-Don"，意为敲击鼓面的声音。
```

File: wiki/Mascots/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/Mascots/zh.md:29]: 没想好改法 但确实有点怪 先按你说的改吧

**Comment by kj415j45** [comment]
pro tip: Switch to `Files changed` tab to use `Add suggestion to batch` feature. This would reduce your commit times and avoid unnecessary rebase:
image

File: wiki/Mascots/zh.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+# 吉祥物
+
+*参见: [吉祥物/画廊](/wiki/Mascots/Gallery)*
+
+这个 YouTube 视频展示了所有的 osu! 吉祥物：[Mascot Showcase](https://youtu.be/mJF2cAs_MrI)
+
+## 官方
+
+### ![osu! icon](/wiki/shared/mode/osu.png) pippi
```
Comment (TPGPL) [inline wiki/Mascots/zh.md:9]: the alt text ("osu! icon") needs to be translated

same for the images below

File: wiki/Mascots/zh.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+特工 J，又称为 BA-2 (节拍特工-2) 或 J，曾于 2008 年作为 osu! 吉祥物的一员，但随后于 2014 年被解雇。他的艺术形象也由 iNiS 公司设计，并也曾作为老版官方网页的一部分。
+
+特工 J 也曾出现在 [LuigiHann](https://osu.ppy.sh/users/1079) 设计的皮肤 [Elite Beat osu! HD (1.0 Complete!)](https://osu.ppy.sh/community/forums/topics/190357) 中。
+
+### Don
+
+![Don](img/Don.png "Don")
+
+> Don 是游戏 [太鼓达人](https://zh.wikipedia.org/wiki/%E5%A4%AA%E9%BC%93%E4%B9%8B%E9%81%94%E4%BA%BA%E7%B3%BB%E5%88%97) 系列的主角。他是一只有白色鼓边、红橙色鼓面、淡蓝色身体、四只脚的太鼓。Don 的梦想是把太鼓的美妙带给全世界。在他搬到和田屋三年后，城镇中无人不晓他的名号。他食量惊人，经常花重金买许多好吃的，以至于和田屋有时候都负担不起他的消费。他的口癖是经常在句尾带上一句 "Ta-Don"，意为敲击鼓面的声音。
```
Comment (TPGPL) [inline wiki/Mascots/zh.md:67]:
```suggestion
> Don 是游戏 [太鼓达人](https://zh.wikipedia.org/wiki/太鼓之達人系列) 系列的主角。他是一只有白色鼓边、红橙色鼓面、淡蓝色身体、四只脚的太鼓。Don 的梦想是把太鼓的美妙带给全世界。在他搬到和田屋三年后，城镇中无人不晓他的名号。他食量惊人，经常花重金买许多好吃的，以至于和田屋有时候都负担不起他的消费。他的口癖是经常在句尾带上一句 "Ta-Don"，意为敲击鼓面的声音。
```

File: wiki/Mascots/zh.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+*关于新闻帖子，参见：[The new osu!taiko mascot is here!](https://osu.ppy.sh/home/news/2017-05-25-the-new-osutaiko-mascot-is-here)*
+
+![Mocha](img/Mocha.png "Mocha")
+
+Mocha 是 osu!taiko 游戏模式的官方吉祥物。她最初由 [Crowie](https://osu.ppy.sh/users/6894067) 在 [sixth fanart contest](https://osu.ppy.sh/community/contests/2) 中设计，并荣获比赛的第 21 名。
+
+## 客串
+
+### 一本木龙太
```
Comment (TPGPL) [inline wiki/Mascots/zh.md:43]:
```suggestion
### Ryūta Ippongi
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**